### ISSUE TITLE
define with multiple has changed meaning in new parser

### ISSUE THREAD

**Issue by eivindjahren** [equinor/ert#5894]
Adding `DEFINE <SUMMARY_VECTORS> FOPT FPR` to the config has changed meaning in relations
to how `CSV_EXPORT2` workflow job works in a workflow file.

Say you had the following in your workflow file:

```
CSV_EXPORT2 <SUMMARY_VECTORS>
```

Before, `FOPT` and `FPR` would be interpreted as individual keys. In the new version ert:

```
.../python3.8/site-packages/ecl/summary/ecl_sum.py
raise ValueError("No valid key")
```

because it is looking for the summary vector "FOPT FPR"

**Comment by eivindjahren** [comment]
The change was introduced by https://github.com/equinor/ert/commit/6b4023936542b746027fd3b584772e3a210c1c71 .

**Comment by oysteoh** [comment]
@oyvindeide / @sondreso  this is a breaking change ( i think released in 2023.06 - correct me if i'm wrong @eivindjahren ) . Should we make an announcement for the users / add it to the release docs / post a message on slack... ?

**Comment by eivindjahren** [comment]
Correct, adding to docs and the next announcement should be the right course of action.
